src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -427,6 +435,17 @@ public void LongRunningResponseDefined()
             var messages = ValidateSwagger(Path.Combine("Resource","Swagger", "Validation", "positive", "long-running-valid-response.json"));
             messages.AssertOnlyValidationMessage(typeof(LongRunningResponseValidation), 0);
         }
+
+        /// <summary>
+        /// Verifies that a providers path is of proper format
+        /// </summary>
+        [Fact]
+        public void ProvidersPathValidJson()
+        {
+            var messages = ValidateSwagger(Path.Combine("Resource", "Swagger", "Validation", "positive", "providers-path-valid.json"));
```

Comment:
I understand I am being inconsistent here but not all paths in this file have ```Core.Utilities.Extensions.CodeBaseDirectory``` just wondering why that is so